src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -378,6 +379,20 @@ public async Task SendLicenseExpiredAsync(IEnumerable<string> emails, string org
         await _mailDeliveryService.SendEmailAsync(message);
     }
 
+    public async Task SendRequestSMAccessToAdminEmailAsync(IEnumerable<string> emails, string organizationName, string requestingUserName, string emailContent)
+    {
+        var message = CreateDefaultMessage("Access Requested for Secrets Manager", emails);
+        var model = new RequestSecretsManagerAccessViewModel
+        {
+            OrgName = CoreHelpers.SanitizeForEmail(organizationName, false),
+            UserNameRequestingAccess = CoreHelpers.SanitizeForEmail(requestingUserName, false),
+            EmailContent = CoreHelpers.SanitizeForEmail(emailContent),
```

Comment:
There has been problems with double-HTML escaping in the past.

We should either use `{{` in the handlebars' template + `.SanitizeForEmail(<target>, false)`.

or 

`{{{` + `.SanitizeForEmail(<target>)`.

My recommendation is to let handlebars handle HTML sanitization via `{{` method.